StudentsMVC/WebContent/WEB-INF/pages/index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
```

Comment:
Please apply proper formatting (mainly indentation) to this markup file.

Reply:
Not sure why it is not formated as in the IDE it is (I'm using Ctrl+Shift+F before every Save). However I've updated and formated the document again, hope it will be properly commited.